backend/app/main/schemas.py

```diff
@@ -118,3 +119,20 @@ class Token(TokenBase):
 
     class Config:
         orm_mode = True
+
+
+class TrayCollectBase(BaseModel):
+    pickup_time: datetime
+    vendor_id: int
+
+
+class TrayCollectCreate(TrayCollectBase):
+    pass
```

Comment:
great job! Just a quick question, I see you're using the TrayCollectCreate schema in the route "/add_tray_collect, why is this just pass?

Reply:
I am not sure to understand, it has always been like this if you look the whole file. The meaning is that it will take in consideration pickup_time and vendor_id.